src/foam/nanos/logger/DiscardLogger.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.logger',
+  name: 'DiscardLogger',
+  implements: [ 'foam.nanos.logger.Logger' ],
+  extends: [ 'ProxyLogger' ],
```

Comment:
so this is just 
extends: foam.nanos.logger.ProxyLogger,
place if after name:

Reply:
Not sure what you mean by - "place if after name:"
Assuming it was a typo for 'it'